mk/uvmt/uvmt.mk

```diff
@@ -86,7 +86,7 @@ SIM_RESULTS             ?= $(if $(CV_RESULTS),$(abspath $(CV_RESULTS))/$(SIMULAT
 SIM_CFG_RESULTS          = $(SIM_RESULTS)/$(CFG)
 SIM_COREVDV_RESULTS      = $(SIM_CFG_RESULTS)/corev-dv
 SIM_LDGEN_RESULTS        = $(if $(LDGEN),$(SIM_CFG_RESULTS)/$(LDGEN),$(SIM_CFG_RESULTS))
-SIM_TEST_RESULTS         = $(SIM_CFG_RESULTS)/$(TEST)
+SIM_TEST_RESULTS         = $(SIM_CFG_RESULTS)/$(TEST_NAME)
```

Comment:
AFAIKT, this is the only use of the variable `$TEST_NAME`.   How does it work?

Reply:
The `$(TEST_NAME)` variables is already in used multiple places, even inside the general test target: https://github.com/openhwgroup/core-v-verif/blob/cv32e40x/dev/mk/uvmt/xrun.mk#L436

The TEST_NAME value is the simplified name of the test which is derived from the required `name` member inside of the test yaml file.